app/src/userpages/components/StreamPage/Show/SecurityView/index.jsx

```diff
@@ -155,6 +155,7 @@ export function SecurityIcon({
 }
 
 export function SecurityView(props: Props) {
+    // $FlowFixMe `updateEditStream` not in OwnProps or StateProps.
```

Comment:
@mondoreale any idea what the problem is here

Reply:
No idea. That's the main "wtf" here. I left those out.